src/main/java/com/launchcode/recipeproject/controllers/HomeController.java

```diff
@@ -29,7 +29,9 @@ public String displayIndex(Model model, Principal principal){
 
         // example of how to bring a user in from an authenticated session
         if (principal != null){
+            System.out.println(principal.getName());
```

Comment:
Im guessing this is just for testing? if so should we add a TODO comment to remind us to delete it when we are in a more functional state